boost-maven/boost-boms/booster-ee7-bom/pom.xml

```diff
@@ -43,7 +43,7 @@
 				<!-- Booster version distinct from the BOM, boost-maven-plugin version -->
 				<!-- Could also use 7.0, 8.0 for EE7, 8 -->
 				<!-- Probably will be 1.0 for initial GA -->
-				<version>0.1-SNAPSHOT</version>
+				<version>2.0-M1-SNAPSHOT</version>
```

Comment:
Conventionally, I believe the naming should be like `2.0.M1-SNAPSHOT`, not `2.0-M1-SNAPSHOT`. That way the final released version looks like `2.0.M1`. At least that is how I have seen it in the past, and how I versioned it in the LMP 3.0.M1 release. Maybe @scottkurz can confirm if we want to proceed with this change across the board?

Reply:
I don't have a strong opinion, and in reading 
https://maven.apache.org/pom.html#Version_Order_Specification
it seems we have a lot of flexibility and yet can still end up with a well-defined ordering scheme.

Maybe it depends what we are tracking exactly, JAX-RS API versions vs. Liberty feature versions.   If we say we're tracking Java/Jakarta EE versions which might have an **x.y.z** version name then I could see using (-) to indicate where the version transitions from "theirs" to "ours"...  so more like Andy's existing `2.0-M1-SNAPSHOT`. 

But if there were other conventions maybe that'd be helpful.  

Also I know we threw out the "M1" idea.. but it might be more suggestive to keep this as "alpha" for now... so 2.0-a1-SNAPSHOT or 2.0-alpha-1-SNAPSHOT .    

What do you think?

Reply:
Hmm..so for e.g., CDI 2.0 the latest commit uses:
`2.0.0.M1-SNAPSHOT`
Charles' suggestion was:  `2.0.M1-SNAPSHOT` and mine was `2.0-a1-SNAPSHOT`.  

I think either is fine... but I don't want to add an extra .0, like `2.0.0.... ` . I haven't digested every point in https://maven.apache.org/pom.html#Version_Order_Specification  but my reading is that as long as we're consistent going forwards, either would work. 

We could figure out the M1 vs a1 later than this PR.

Reply:
BTW, looks like the Travis failure https://api.travis-ci.org/v3/job/567141295/log.txt shows you need to still change the version in the boosters and the IT poms.